scenario/auto_map.yml

```diff
@@ -2,11 +2,9 @@ input:
     input.go: |
```

Comment:
Please revert all changes to the scenarios, and create one new scenario called directive.yml with this content, this should cover all cases:
```go

//goverter:converter
type Simple interface {
	//goverter:context c
	//goverter:map Value | FromContext
	Convert(source struct{ Value int }, c string) struct{ Value string }
}

type (
	//goverter:converter
	One interface{}
	Two interface{}
)

//goverter:variables
var (
	Convert func(string) string
)

//goverter:context c
func FromContext(c string) string { return c }
```